model_specific/SM/decays/standard_model_decay_table.cpp

```diff
@@ -0,0 +1,81 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+
+#include "standard_model_decay_table.hpp"
+
+#include "error.hpp"
+
+#include <iostream>
+#include <sstream>
+
+namespace flexiblesusy {
+
+standard_model_decay_table::standard_model_decay_table()
+   : decay_table({Decays_list(25)})
+{
+}
+
+void standard_model_decay_table::clear()
+{
+   for (auto& particle: decay_table) {
+      particle.clear();
+   }
+}
+
+/*
+void SM_decay_table::print(std::ostream& ostr) const
```

Comment:
Why is this function commented out? Some printing function would be very useful, I think, so we may keep it (in this or another form).

Reply:
Uncommented in 6937ae2.

Reply:
Thanks for this!